python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -250,7 +251,7 @@ def check(self, args: SimpleNamespace) -> None:
             # Passing the same inputs that were used to initialize the output
             # collection is allowed; this means they must _end_ with the same
             # collections, because we push new runs to the front of the chain.
-            for c1, c2 in zip(self.inputs[::-1], self.output.chain[::-1]):
+            for c1, c2 in zip(self.inputs[::-1], self.output.chain[::-1], strict=True):
```

Comment:
This one also breaks everything...

Reply:
I think `strict=False` is the right thing to do here.